Controllers/StaffController.cs

```diff
@@ -12,34 +13,26 @@ namespace stunning_robot_HR.Controllers
 {
     public class StaffController : Controller
     {
-        private readonly stunning_robot_HRContext _context;
-
-        public StaffController(stunning_robot_HRContext context)
+        private stunning_robot_HRContext _context;
+        private  IStaffRepository staffRepository;
+        public StaffController(stunning_robot_HRContext context )
```

Comment:
create a parameter for your constructor for IStaffRepository, and let the Dependency Injection handle the creation of the object for you.